docs/software/python.md

```diff
@@ -201,7 +195,7 @@ Sometimes conda environments can "break" if two packages in the environment requ
 * Reinstall the packages all within the same install command (e.g., `conda install <package1> <package2>`). This forces conda to attempt to resolve shared library conflicts. 
 * Create a new environment and reinstall the packages you need (preferably installing all with the same `conda install` command, rather than one-at-a-time, in order to resolve the conflicts).
 
-## Mamba
+## Mamba Package Manager
 
 [Mamba](https://mamba.readthedocs.io/) is a fast, robust, and cross-platform package manager that aims to be a drop-in replacement for _conda_. Utilizing Mamba can improve the speed and reliability of constructing an environment. To use Mamba on an Alpine or Blanca compute node, perform the following module load: 
```

Comment:
I think it would be nice, though not absolutely necessary, to include some specific use cases. e.g. Mamba can be useful for installing large conda environments and/or environments that failed to install with conda.